optuna/integration/keras.py

```diff
@@ -12,18 +12,17 @@
 if not _imports.is_successful():
     Callback = object  # NOQA
 
+_keras_pruning_callback_deprecated_msg = "Recent Keras release (2.4.0) simply redirects "
+"all APIs in the standalone keras package to point to tf.keras. "
+"There is now only one Keras: tf.keras. "
+"There may be some breaking changes for some workflows by upgrading to keras 2.4.0. "
+"Test before upgrading. "
+"REF: https://github.com/keras-team/keras/releases/tag/2.4.0. "
+"There is an alternative callback function that can be used instead: "
+":class:`~optuna.integration.TFKerasPruningCallback`"
```

Comment:
```suggestion
_keras_pruning_callback_deprecated_msg = (
    "Recent Keras release (2.4.0) simply redirects "
    "all APIs in the standalone keras package to point to tf.keras. "
    "There is now only one Keras: tf.keras. "
    "There may be some breaking changes for some workflows by upgrading to keras 2.4.0. "
    "Test before upgrading. "
    "REF: https://github.com/keras-team/keras/releases/tag/2.4.0. "
    "There is an alternative callback function that can be used instead: "
    ":class:`~optuna.integration.TFKerasPruningCallback`"
)
```

String concatenation would be nice to be enclosed in parentheses.